internal/rpcproxy/rpcproxylogin_test.go

```diff
@@ -173,6 +173,37 @@ func TestProxySocketsAdminFacade(t *testing.T) {
 			ErrorCode: "unauthorized access",
 		},
 		oauthAuthenticatorError: errors.E(errors.CodeUnauthorized),
+	}, {
+		about: "login with username/password fails",
+		messageToSend: rpcproxy.Message{
+			RequestID: 1,
+			Type:      "Admin",
+			Version:   3,
+			Request:   "Login",
+			Params:    []byte(`{"auth-tag":"user-bob"}`),
```

Comment:
Why is the auth-tag user-bob, why not user-admin?

Reply:
No reason, it could be anything.